docs/helm-deployment-aws_cloud.md

```diff
@@ -328,6 +328,49 @@ export EFS_SERVER=$EFS_FS_ID.efs.$AWS_DEFAULT_REGION.amazonaws.com
 # Publish EFS Volume's DNS name (if not already done)
 export EFS_SERVER=<EFS_ID>.efs.<AWS-REGION>.amazonaws.com
 ```
+### Create a Docker registry pull secret
+
+Since we need to use protected Docker images from Quay.io, you need to access to a secret with credentials to be able to pull those images.
+
+* Log in to Quay.io with your credentials:
+```bash
+docker login quay.io
+```
+
+* Generate a base64 value for your dockercfg. This will allow Kubernetes to access Quay.io:
+```bash
+cat ~/.docker/config.json | base64 (Linux)
+base64 -w 0 ~/.docker/config.json  (Windows)
+```
+
+* Create a file `secrets.yaml` file and insert the following content:
+```bash
+apiVersion: v1
+kind: Secret
+metadata:
+  name: quay-registry-secret
+type: kubernetes.io/dockerconfigjson
+data:
+# Docker registries config json in base64 to do this just run:
+# cat ~/.docker/config.json | base64
+  .dockerconfigjson: add-your-base64-string
+```
+
+* Add the base64 string generated in the previous step to ``.dockerconfigjson``.
+
+* Deploy the secret into the kubernetes cluster
+```bash
+kubectl create -f secrets.yaml --namespace $DESIREDNAMESPACE
+```
+* You should see the following output:
+```bash
+secret "quay-registry-secret" created
+```
+
+* When installing the ACS Helm chart, add a variable:
+```bash
+--set registryPullSecrets=quay-registry-secret
```

Comment:
Also add at line 447 - example command for Deploying Alfresco Content Services below

Reply:
It should be added on each example command (there are 4 examples in this doc)